code/modules/admin/admin_verbs.dm

```diff
@@ -108,7 +108,8 @@ var/list/admin_verbs_ban = list(
 	/client/proc/jobbans,
 	/client/proc/ban_panel,
 	/client/proc/BCCM_toggle,
-	/client/proc/BCCM_WhitelistPanel
+	/client/proc/BCCM_WhitelistPanel,
+	/client/proc/cmd_admin_cryo
```

Comment:
it checks R_MOD but is in the +BAN verb list
either make it check R_BAN or put it in admin_verbs_mod